test/Microsoft.ML.Tests/Scenarios/Api/CookbookSamples/CookbookSamples.cs

```diff
@@ -149,7 +149,7 @@ private void TrainRegression(string trainDataPath, string testDataPath, string m
             using (var stream = File.Create(modelPath))
             {
                 // Saving and loading happens to 'dynamic' models, so the static typing is lost in the process.
-                mlContext.Model.Save(model.AsDynamic, stream);
+                mlContext.Model.Save(model, stream);
```

Comment:
>stream [](start = 44, length = 6)

Note that when we update these samples in the `CookbookSamples.cs`, there is usually a section in the actual cookbook that needs to be updated. (In this case I think [it is this one](https://github.com/dotnet/machinelearning/blob/9caafb04d2b4f298f2a9479a9550597afee32c94/docs/code/experimental/MlNetCookBookStaticApi.md#how-do-i-save-and-load-the-model).)

Reply:
fixed